dashboard/test/ui/features/star_labs/dropdown.feature

```diff
@@ -1,15 +1,15 @@
 Feature: Dropdowns work as expected
 
 Background:
-  Given I am on "http://studio.code.org/flappy/1?noautoplay=true"
```

Comment:
It feels like chances are low that we make edits to 'sports' in the future, but is there a downside in having 'noautoplay=true' on the new level?

Reply:
fwiw, we use it [a lot](https://github.com/code-dot-org/code-dot-org/search?l=Gherkin&q=noautoplay) in UI tests.